pandas/tests/frame/test_reshape.py

```diff
@@ -918,6 +918,90 @@ def test_unstack_swaplevel_sortlevel(self, level):
         tm.assert_frame_equal(result, expected)
 
 
+def test_explode():
+    # GH 16538
+
+    # Automatically do str.split
+    columns = ['a', 'b', 'c']
+    df = pd.DataFrame([['foo,bar', 'x', 42],
+                       ['fizz,buzz', 'y', 43]],
+                      columns=columns)
+    rs = df.explode('a', sep=',')
```

Comment:
Also use `result` and `expected` here instead of `rs` and `xp`

Reply:
I think in the current context `rs` and `xp` has clear meaning and is more concise to read. IMO it's the same as naming a variable `df` instead of `dataframe`.

Reply:
`result` and `expected` are practically the standard. You'll see that in other tests in the module / larger code base so best to stay consistent. Will appear less verbose once parametrized

Reply:
I agree that consistency is good. However, I see a lot of different variations throughout the code base on tests (r, res, rs, result, results, res*, xp, e, xp, exp, exp*, expected, etc). In addition, other abbreviations throughout the codebase are equally clear in context:

dataframe -> df
series -> s
index -> idx
columns -> cols
number_of_something -> n
operator -> op
function -> fun

Reply:
`result` and `expected` should far and away be the most widely used. While not 100% there using `rs` and `xp` is moving away from consistency so please change

Reply:
I don't see reviewers up in arms about `fun` v `f` to denote a function, or `n` vs `num`, or `s` vs `ser`.  Not everything needs to have a strictly enforced standard.

Reply:
I am ok with the naming as long as it follows the module where the tests are. We are somewhat inconsistent across test modules. But in a single module should be consistent with the existing style.

We for sure prefer fully written out names: 
result, expected, df, op, s are ok.
idx and cols are generally not. res, and xp are generally not